addons/web/static/src/fields/email_field.xml

```diff
@@ -3,7 +3,7 @@
 
     <t t-name="web.EmailField" owl="1">
         <t t-if="props.readonly">
-            <a class="o_form_uri o_text_overflow" t-att-href="'mailto:'+props.value" t-esc="props.value"/>
+            <a class="o_form_uri o_text_overflow" t-att-href="'mailto:'+props.value" t-esc="props.value || ''"/>
```

Comment:
t-att-href if value is falsy?

Reply:
yes, from the model point of view, false is a legitimate value, meaning "unset".
Domains need to be able to make the difference between unset and empty string.
But, at display time, false.toString gives "false", and we want the value to be ""